numpy-cheatsheet/cheatsheet_prompt.py

```diff
@@ -0,0 +1,38 @@
+# cheatsheet_prompt.py
```

Comment:
Again, I'm guessing you're not using this script if you're swapped to using Groq. 

If you want to figure out API access to Groq and then algorithmically create the cheatsheets then please update the script.  I'd provide more comments on the code below, but based on our conversations, I don't think you're using this code at all.  You can remove it from the PR.